src/main/java/net/library/controller/UserController.java

```diff
@@ -25,14 +28,30 @@ public UserController(final UserService service) {
         this.service = service;
     }
 
-    @PostConstruct
-    public void initUserList() {
-        service.initUserLst();
+    @GetMapping()
+    public ResponseEntity<List<User>> getAllUsers() {
+
+        return new ResponseEntity<>(service.getAllUsers(),HttpStatus.OK);
     }
 
-    @GetMapping()
-    public ResponseEntity<List<User>> getUsers() {
+    @PostMapping("/create")
```

Comment:
use please best practices for rest api naming (https://restfulapi.net/resource-naming/)